spec/features/post_peep_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require_relative './helpers/sign_up.rb'
```

Comment:
these are not indented correctly and I think they'd be better off in spec_helper.rb